harvard-lts/CURIOSity
# Rotated keys

**New Creds enc file**
* * *

# What does this Pull Request do?
As part of the LPE rotation, I generated a new master key and credentials enc file. This PR is on curiosity-dev which was deployed in conjunction with the new IF.

# How should this be tested?

To test, you need to run in production mode. Put the new key in your env file and make sure the site can come up. You can also confirm the pair works together by going into your rails container and running the following:
EDITOR="vi" bin/rails credentials:edit

The file being unlock able proves its all set.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
